Topics/Tech_Stacks.md

```diff
@@ -1 +1,47 @@
 ## Tech Stacks
+
+---
+
+# Access to Salesforce API 
```

Comment:
Please create a Salesforce specific file and move these there! If you are borrowing from a page, you can use a link to the page rather than repeating the content here

Reply:
Would the salesforce specific file in the Topics folder?